### PR TITLE
Cosmetics

### PR DESCRIPTION
**Pull request by bencbernstein**

### CODE REVIEW

File: src/Components/DaisyChainAnimation/index.js [hunk 1]
```diff
@@ -9,11 +9,26 @@ class DaisyChainAnimation extends Component {
...
+{ value: 'pterodactyl', definition: 'an extinct flying dinosaur which had a finger on its wing', indices: [0,4], marginAdjustment: '23%' },
+{ value: 'quadruped', definition: 'a four legged animal', indices: [], marginAdjustment: '25%' },
+{ value: 'quadrilateral', definition: 'a four sided shape', indices: [0,3], marginAdjustment: '18%' },
+{ value: 'equilateral', definition: 'having equal sides', indices: [4,10], marginAdjustment: '23%' },
+{ value: 'polydactylic', definition: 'having many or too many fingers or toes', indices: [], marginAdjustment: '20%' },
+{ value: 'polyhedron', definition: 'a solid geometrical shape with many faces', indices: [0,3], marginAdjustment: '24%' },
+{ value: 'protoplasm', definition: 'the living material inside a cell, so named because it is the first formed thing', indices: [], marginAdjustment: '24%' },
+{ value: 'isometric', definition: 'having equal measure', indices: [], marginAdjustment: '25%' },
+{ value: 'lithograph', definition: 'a symbol or inscription on stone', indices: [0,3], marginAdjustment: '24%' },
+{ value: 'monolith', definition: 'a single piece of stone	', indices: [4,7], marginAdjustment: '27%' },
```
Comment (OLI9292) [inline src/Components/DaisyChainAnimation/index.js]: extra space in the definition?

File: src/Components/DaisyChainAnimation/index.js [hunk 2]
```diff
@@ -9,11 +9,26 @@ class DaisyChainAnimation extends Component {
...
+{ value: 'monogram', definition: 'a symbol with one initial letter from each word', indices: [4,7], marginAdjustment: '26%' },
+]
```
Comment (OLI9292) [inline src/Components/DaisyChainAnimation/index.js]: interesting solution w/ the marginAdjustment? does it work to good effect?

File: src/Components/DaisyChainAnimation/index.js [hunk 3]
```diff
@@ -9,11 +9,26 @@ class DaisyChainAnimation extends Component {
...
-      { value: 'cryptography', definition: 'the writing and decoding of secret codes', indices: [0,4] },
-      { value: 'cryptogram', definition: 'a piece of writing in a secret code', indices: [0,4] },
-      { value: 'monogram', definition: 'a symbol with one initial letter from each word', indices: [4,7] },
-      { value: 'lithophyte', definition: 'a plant that grows upon stone, or a plant that is stony in substance, such as coral', indices: [0,4] }
-    ]
+      { value: 'cryptogram', definition: 'a piece of writing in a secret code', indices: [0,4], marginAdjustment: '25%' },
+{ value: 'cryptography', definition: 'the writing and decoding of secret codes', indices: [0,4], marginAdjustment: '20%' },
+{ value: 'telegraph', definition: 'a device for sending a written message over a great distance by wires', indices: [4,8],marginAdjustment: '25%' },
+{ value: 'dermatoglyph', definition: 'the fingerprint, i.e. a symbol found in the skin of the fingertips', indices: [], marginAdjustment: '20%' },
+{ value: 'decagon', definition: 'a geometrical shape with ten angles', indices: [], marginAdjustment: '29%' },
```
Comment (OLI9292) [inline src/Components/DaisyChainAnimation/index.js]: are some of these missing indices purposefully?

File: src/Components/DaisyChainAnimation/index.js (hunk 2 above)
Comment (bencbernstein) [reply src/Components/DaisyChainAnimation/index.js]: yeah it's a big improvement. even better would be counting characters and dividing by a constant but this is good for now.

File: src/Components/DaisyChainAnimation/index.js (hunk 3 above)
Comment (bencbernstein) [reply src/Components/DaisyChainAnimation/index.js]: yeah, some don't daisy chain

**Comment by OLI9292** [comment]
cool, lets merge and deploy

**Pull request merged**